packages/py2_wheel.rb

```diff
@@ -0,0 +1,26 @@
+require 'package'
+
+class Py2_wheel < Package
+  description 'Wheel is the binary package format for python.'
+  homepage 'https://wheel.readthedocs.io/'
+  @_ver = '0.36.2'
+  version @_ver
+  license 'MIT'
+  compatibility 'all'
+  source_url 'https://github.com/pypa/wheel.git'
+  git_hashtag @_ver
```

Comment:
No binaries?

Reply:
It builds almost instantaneously

Reply:
Well, if there are no binaries, we need a way to let those would build binaries know to ignore this package.  That is done by adding exclusions to `tools/needs_binaries.sh`.  If you don't want to spend the time to update that script then it doesn't hurt to generate binaries.